framework/src/modules/token/events/recover.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { BaseEvent, EventQueuer } from '../../base_event';
+import {
+	TOKEN_ID_LENGTH,
+	TokenEventResult,
+	TokenErrorEventResult,
+	CHAIN_ID_LENGTH,
+} from '../constants';
+
+export interface RecoverEventData {
+	terminatedChainID: Buffer;
+	tokenID: Buffer;
+	amount: bigint;
+	recipientAddress: Buffer;
```

Comment:
[LIP0051 in branch update-lip0051-token-module](https://github.com/LiskHQ/lips/blob/update-lip0051-token-module/proposals/lip-0051.md#initializeescrowstore) doesn't refer `recipientAddress` field. Guess it can be removed from `RecoverEventData` interface.